spec/models/course_group_spec.rb

```diff
@@ -27,7 +27,7 @@
         allow(CourseGroups::Leadership).to receive(:new).with(course_group: subject, cohort:, schedule_date:).and_return(service)
         expect(CourseGroups::Leadership).to receive(:new).with(course_group: subject, cohort:, schedule_date:)
 
-        expect(subject.schedule_for(cohort:, schedule_date:)).to eq(true)
+        expect(subject.schedule_for(cohort:, schedule_date:)).to be_truthy
```

Comment:
I was wondering for those and anywhere we're using `eq(true)` / `eq(false)` we can use `be true` / `be false` instead as we are checking explicitly for booleans?